app/views/groups/index.html.erb

```diff
@@ -0,0 +1,12 @@
+
+<section>
+<h1>Categories</h1>
+
+<div id="groups" class="categories">
+  <% @groups.each do |group| %>
+    <%= render group %>
+  <% end %>
+  <%= link_to "Add New Category", new_group_path, class: "add-category btn btn__sign-in" %>
+</div>
+
+</section>
```

Comment:
- [ ] The button is at the bottom of the page in the design, kindly fix this
in your design :point_down: 
![Screenshot from 2023-06-01 11-34-23](https://github.com/gsmalik030/my-budget-app/assets/66097371/da612d75-31d7-4284-a606-c3051d3c3923)
given design :point_down: 
![Screenshot from 2023-04-28 11-55-45](https://github.com/gsmalik030/my-budget-app/assets/66097371/5f7485f1-b1d4-44f9-9e62-579399fffdfe)